types/matter-js/index.d.ts

```diff
@@ -1385,11 +1385,20 @@ declare namespace Matter {
          * Triggers `beforeRemove` and `afterRemove` events on the `composite`.
          * @method remove
          * @param {Composite} composite
-         * @param {any} object
+         * @param {Body | Composite | Constraint | MouseConstraint | (Body | Composite | Constraint | MouseConstraint)[]} object
```

Comment:
I did expect that it would allow in jsdoc to have no type defined in the param, but I guess not? 
But this is fine too, but prefer to align the jsdoc type to the type set in the ts (Array instead of []). Bu this is optional.

```suggest
         * @param {Body | Composite | Constraint | MouseConstraint | Array<Body | Composite | Constraint | MouseConstraint>} object
 ```